src/utils/format.ts

```diff
@@ -9,3 +9,7 @@ export function formatDate(date: Date) {
 export function formatDateRange(date1: Date, date2: Date) {
   return formatDate(date1) + ' - ' + formatDate(date2);
 }
+
+export const formatNumber = (value: number) => {
+  return Intl.NumberFormat('en-US').format(value);
```

Comment:
It's a lot more performant to instantiante `Intl.NumberFormat` only once outside of the function, and use this single instance inside the function.
Indeed the constructor of `Intl.NumberFormat` does a ton of things.

Reply:
Oh alright. The same way the `dateFormatter` was implemented.

Reply:
I couldn't use the function here https://github.com/Netacci/perfcompare/blob/184dfb1bfcddc121399c129668ddd0cdcd0440ad/src/components/CompareResults/RevisionRowExpandable.tsx#L111 because `medianDifference` is a string. Was that done deliberately?